integration_tests/configs/default.jsonnet

```diff
@@ -10,6 +10,7 @@
     },
     'app-config': {
       'app-db-backend': 'rocksdb',
+      'db-backend': 'rocksdb',
```

Comment:
@yihuang actually we seems miss this config, since `db-backend` [overwrite](https://github.com/cosmos/cosmos-sdk/blob/fe34e5df3257e458dcfb3a1d634d812437fed553/server/util.go#L367) `app-db-backend`